server/http_handler.go

```diff
@@ -147,7 +148,13 @@ func (t *tikvHandlerTool) getMvccByEncodedKey(encodedKey kv.Key) (*kvrpcpb.MvccG
 		},
 	}
 	kvResp, err := t.store.SendReq(tikv.NewBackoffer(context.Background(), 500), tikvReq, keyLocation.Region, time.Minute)
-	log.Info(string(encodedKey), keyLocation.Region, string(keyLocation.StartKey), string(keyLocation.EndKey), kvResp, err)
+	logutil.Logger(context.Background()).Info("get MVCC by encoded key",
+		zap.String("encodeKey", string(encodedKey)),
```

Comment:
It is better to use zap.Binary to print a key.

Reply:
Done.